bin/main.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "../lib/game.rb"
+
+# Entry point for the program -> Game
+puts "Welcome to tic-tac-toe"
+puts "Instructions: The boxes are numbered from 1 to 9"
+puts "A player has to input a number between 1 to 9"
+puts "Once input, it will display in its appropriate box with letters either X or O"
+puts "May the best player win, GoodLuck!"
+puts "  You enter the grid number you want to play your piece at."
+print "\n\n"
+
+
+print "Enter name player1: "
```

Comment:
The same way you are asking for and prompting in this file you can do the same with some of the chomps and puts statements in your `Game` class.

Where you get the info you need from a user and call a method based on what you've prompted for.


***Please note: I'm not asking you to transfer `Game` class to main but simply add the scripting logic to main and call Game methods in your main script file.
This would help you improve the way you code. It is generally a bad idea to have `puts` and `get` statements in a class. Our project itself needs some refactoring in that regards 😅***